doc/specs/stdlib_stats_distribution_normal.md

```diff
@@ -0,0 +1,262 @@
+! 0.414566994  3.06084275  1.86505437  1.36338580  7.26878643E-02
+! 0.178585172  1.39557445  0.828021586  0.872084975
+
+    loc = (-1.0, 2.0)
+    scale = (2.0, 1.0)
+    print *, norm(loc, scale)
+    !single complex normal random variate with real part of mu=-1, sigma=2;
+	!imagainary part of mu=2.0 and sigma=1.0
+
+! (1.22566295,2.12518454)
+
+end program demo_normal_rvs
+```
+
+## `pdf_normal` - normal distribution probability density function
+
+### Status
+
+Experimental
+
+### Description
+
+The probability density function of the continuous normal distribution.
+
+$$f(x) = \frac{1}{\sigma \sqrt{2}} e^{-\frac{1}{2}(\frac{x-\mu}{\sigma})^{2}}$$
+
+### Syntax
+
+`result = [[stdlib_stats_distribution_normal(module):pdf_normal(interface)]](x, loc, scale)`
+
+### Class
+
+Elemental function
+
+### Arguments
+
+`x`: has `intent(in)` and is a scalar of type `real` or `complex`.
+
+`loc`: has `intent(in)` and is a scalar of type `real` or `complex`.
+
+`scale`: has `intent(in)` and is a scalar of type `real` or `complex`.
+
+All three arguments must have the same type.
+
+### Return value
+
+The result is a scalar or an array, with a shape conformable to arguments, of type `real`.
+
+### Example
+
+```fortran
+program demo_normal_pdf
+    use stdlib_random, only : random_seed
+    use stdlib_stats_distribution_normal, only : norm_pdf=>pdf_normal, &
+                                                 norm => rvs_normal
+
+    implicit none
+    real :: x(3,4,5),a(3,4,5),b(3,4,5)
+    complx :: loc, scale
+    integer :: seed_put, seed_get
+
+    seed_put = 1234567
+    call random_seed(seed_put, seed_get)
+
+    print *, norm_pdf(1.0,0.,1.) !a probability density at 1.0 in standard normal
+
+! 0.241970733
+
+    print *, norm_pdf(2.0,-1.0, 2.0)
+    !a probability density at 2.0 with mu=-1.0 sigma=2.0
+
+!6.47588000E-02
+
+    x = reshape(norm(0.0, 1.0, 60),[3,4,5])
+    ! standard normal random variates array
+
+    a(:,:,:) = 0.0
+    b(:,:,:) = 1.0
+    print *, norm_pdf(x, a, b)  ! standard normal probability density array
+
+!  0.340346158  0.285823315  0.398714304  0.391778737  0.389345556
+!  0.364551932  0.386712372  0.274370432  0.215250477  0.378006011
+!  0.215760440  0.177990928  0.278640658  0.223813817  0.356875211
+!  0.285167664  0.378533930  0.390739858  0.271684974  0.138273031
+!  0.135456234  0.331718773  0.398283750  0.383706540
+
+    loc = (1.0, -0.5)
+    scale = (1.0, 2.)
+    print *, norm_pdf((1.5,1.0), loc, scale)
+    ! a complex normal probability density function at (1.5,1.0) with real part
+	! of mu=1.0, sigma=1.0 and imaginary part of mu=-0.5, sigma=2.0
+
+! 5.30100204E-02
+
+end program demo_normal_pdf
+```
+
+## `cdf_normal` - normal distribution cumulative distribution function
+
+### Status
+
+Experimental
+
+### Description
+
+Cumulative distribution function of the normal continuous distribution
+
+$$F(x)=\frac{1}{2}\left [ 1+erf(\frac{x-\mu}{\sigma \sqrt{2}}) \right ]$$
```

Comment:
Does this directly cover the complex case? In any case it would be good to provide a reference that covers that.